include/oneapi/dpl/pstl/omp/parallel_merge.h

```diff
@@ -39,20 +46,29 @@ __parallel_merge_body(std::size_t __size_x, std::size_t __size_y, _RandomAccessI
         return;
     }
 
-    _RandomAccessIterator1 __xm;
-    _RandomAccessIterator2 __ym;
-
     if (__size_x < __size_y)
     {
-        __ym = __ys + (__size_y / 2);
-        __xm = std::upper_bound(__xs, __xe, *__ym, __comp);
+        _RandomAccessIterator2 __ym = __ys + (__size_y / 2);
+        __make_parallel_merge_leaf_tasks(::std::upper_bound(__xs, __xe, *__ym, __comp),
```

Comment:
Recently, we came to conclusion that we should not write `::` operator before `std::` in new code, at least.